mysys/my_md5.cc

```diff
@@ -78,32 +78,18 @@ then a md5-hash (16 bytes) string is generated using the complete record.
 Following functions act as helper function in generation of this md5-hash. */
 
 void *wsrep_md5_init() {
-#if defined(HAVE_YASSL)
```

Comment:
This is not a conflict marker resolution. Please mention non trivial changes such as this in the commit message.

Reply:
fixed